cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -376,6 +436,6 @@ output SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT string = {{bicepName
 output SERVICE_BINDING_{{alphaSnakeUpper $name}}_ENDPOINT string = {{bicepName $name}}.properties.vaultUri
 {{end -}}
 {{range $name, $value := .CosmosDbAccounts -}}
-output SERVICE_BINDING_{{alphaSnakeUpper $name}}_ENDPOINT string = {{bicepName $name}}.properties.documentEndpoint
+output connectionStringKey_{{alphaSnakeUpper $name}} string = {{bicepName $name}}.name
```

Comment:
How is the name of the cosmos account the connection string?  I am confused here.